tool/npm_template/node_wrapper.dart

```diff
@@ -66,18 +66,24 @@ external Exports get exports;
 external List<String> _getKeys(Object value);
 
 void main() {
+  const kDebug =
+      const bool.fromEnvironment('GLTF_VALIDATOR_DEBUG', defaultValue: false);
+
+  void onError(Object e, StackTrace st, Function reject) =>
+      kDebug ? reject(st.toString()) : reject(e.toString());
```

Comment:
Sorry to nitpick, one last suggestion here.  The debug mode looks like it hides the actual error message in the process of providing a stack trace.  I think both would be needed.

```
    kDebug ? reject(e.toString() + '\n' + st.toString()) : reject(e.toString());
```

Reply:
StackTrace string representation includes error message.

Reply:
Ah OK